dask/dataframe/io/parquet.py

```diff
@@ -290,9 +296,9 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                              categories=categories, index=index)
 
 
-def to_parquet(path, df, compression=None, write_index=None, has_nulls=None,
+def to_parquet(path, df, compression=None, write_index=None, has_nulls=True,
                fixed_text=None, object_encoding=None, storage_options=None,
-               append=False, ignore_divisions=False):
+               append=False, ignore_divisions=False, partition_on=[]):
```

Comment:
Mutable default keyword args always scare m 😀 though I think partition_on isn't mutated. Still probably best to change it.